src/rest-server/src/models/v2/utils/frameworkConverter.js

```diff
@@ -160,27 +222,78 @@ const convertState = (state, exitCode) => {
   }
 };
 
-const generateExitSpec = (code) => {
-  const exitSpecMap = generateSpecMap();
-  if (!_.isNil(code)) {
-    if (!_.isNil(exitSpecMap[code])) {
-      return exitSpecMap[code];
+const getContainerPorts = (ports, taskIndex, podUID) => {
+  // get container ports
```

Comment:
Why PodUID is changed, but the generated ports are the same.
See:
[https://int-bedrest-server/api/v2/jobs/yqwang~taskhistory-elastic-2/attempts/0/taskRoles/ra/taskIndex/2/attempts](https://int-bedrest-server/api/v2/jobs/yqwang~taskhistory-elastic-2/attempts/0/taskRoles/ra/taskIndex/2/attempts)

Reply:
> Why PodUID is changed, but the generated ports are the same.
> See:
> https://int-bedrest-server/api/v2/jobs/yqwang~taskhistory-elastic-2/attempts/0/taskRoles/ra/taskIndex/2/attempts

Fixed, thanks